quickjs/cutils.c

```diff
@@ -1,6 +1,6 @@
 /*
  * C utilities
- * 
+ *
```

Comment:
Don't format source codes original from quickjs. It makes rebasing new version hard.

Reply:
I didn't format the quickjs code, it's just that my editor automatically removed the extra spaces in the new lines. I suspect that these spaces were introduced by ourselves accidentally, because I checked the quickjs source code and there were no these extra spaces in the original code.